### PR TITLE
Pair programming with your kids

### PR DESCRIPTION
**Pull request by LouisPryer**
### Blog title

Pair programming with your kids. Any other suggestions are welcome

### Blog description

This sparked from a discussion in #twitter around Amazon's STEM toy subscription. Instead of tweeting out the link, I thought we could convert this into an opportunity to recommend some cool resources to help with teaching kids how to code.

#### Author

I admit my grammar and spelling isn't the best, anything I have missed please shout

- [X] I have checked for grammatical errors
- [X] I have checked for spelling errors
- [X] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2017-27-01-pair-programming-with-your-kids.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+STEM(Science Technology Engineering and Mathematics Club) are toys and kits to engage children in learning a variety of topics, ranging from Chemistry, Electronics, Maths and Coding.
+Toys like MakeBlock's [mBot](https://www.amazon.co.uk/mBot-Bluetooth-Version-Scratch-Programmable/dp/B00SK5RUQY) are great hands on projects, this example can be programmed via a developing environment similar to the Arduino environment.
+Although the subscription from Amazon is only available in the USA, they are available elsewhere in the UK. [Amazon](https://www.amazon.co.uk/stem-toys/b?ie=UTF8&node=6714366031), [The Toy Shop](http://www.thetoyshop.com/c/stem-toys) and other stores all have specific pages for STEM toys.
+
+## Gamification
+
+Gamification is the concept of engaging students via games and the method of play.
+This has proven to be very successful, even more so with the growing trend of gaming in recent years.
+There are wide variety of different games and systems to help teach coding to children, here are just a few suggestions some of our engineers in the Holiday Extras web team use to encourage their kids with learning to code.
```
Comment (hx-damian-casey) [inline _posts/2017-27-01-pair-programming-with-your-kids.md]: There are **a** wide variety

**Comment by hx-damian-casey** [comment]
Flows really well dude! 👍 from me once the small changes we discussed are made.

File: _posts/2017-27-01-pair-programming-with-your-kids.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+layout: post
+title: Pair programming with your kids
+categories: Learning to code, Education
+author: louis_pryer
+tags: Education
+comments: true
+excerpt: "Computing and coding are a growing key skill in this day and age to get ahead in the world. However as jobs are hard to get by, when you find them experience and knowledge of the topic are key!"
+---
+
+Computing and coding are a growing key skill in this day and age to get ahead in the world. However as jobs are hard to get by, when you find them experience and knowledge of the topic are key!
```
Comment (steve-at-hx) [inline _posts/2017-27-01-pair-programming-with-your-kids.md]: I think the main argument here is that more tech jobs are being created and are difficult for companies to fill, so a job in tech could be a wise thing to prepare for. There are also more and more ways to use engineering to improve home and work life, even if you're not an engineer. Take home automation.

File: _posts/2017-27-01-pair-programming-with-your-kids.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Although most schools are moving towards computing classes at earlier ages, some students are falling behind due to struggling to understand the material of the course, or that the school has limited time to teach the material of the course.
```
Comment (steve-at-hx) [inline _posts/2017-27-01-pair-programming-with-your-kids.md]: Limited time is the big one, I think. So extra curricular is a great way to keep the interest going.

**Comment by LouisPryer** [comment]
@steve-at-hx perhaps something like this for the introduction?

Computing and coding are a growing key skill in this day and age to get ahead in the world.
With the constant advancement in technology, more and more jobs in the tech sector are being created.
However, as the number of jobs in the tech industry raises, experience and knowledge of the topic are key!
Having engineering skills are also key to improve other aspects of home/work life.
From the raise of Internet of Things smart devices for work/home automation, to budding entrepreneurs creating/hosting their own website to kindle a promising business opportunity.
Although most schools are moving towards computing classes at earlier ages, some students are falling behind due to struggling to understand the material of the course, or that the school has limited time to teach the material of the course.
Some schools have started to introduce extra curriculum activities to offset the time lost in the classroom, but if these aren't available in your school/area, other options are available to encourage your kids.
There are some great tools to help your children out and inspire them to get ahead of the game.
Here are just a few suggestions.

**Pull request merged**